src/components/dropzone/FileItem.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div class="preview-container">
+    <div v-for="file in files" :key="file.name" class="preview-card">
```

Comment:
chỗ này em thấy chưa được ổn.
nếu là đặt tên component là ListFiles thì oke này nhưng nếu là FileItem thì em sẽ viết như thế này sẽ hợp lý hơn.
component FileItem sẽ nhận vào prop là fileType và component này chỉ render ra đúng 1 file còn file nào sẽ dựa vào cái fileType truyền vào
ở ngoài dùng FileType thì v-for nó ra